config/passport.js

```diff
@@ -428,14 +429,15 @@ passport.use('tumblr', new OAuthStrategy({
 /**
  * Foursquare API OAuth.
  */
-passport.use('foursquare', new OAuth2Strategy({
-  authorizationURL: 'https://foursquare.com/oauth2/authorize',
-  tokenURL: 'https://foursquare.com/oauth2/access_token',
-  clientID: process.env.FOURSQUARE_ID,
-  clientSecret: process.env.FOURSQUARE_SECRET,
-  callbackURL: process.env.FOURSQUARE_REDIRECT_URL,
-  passReqToCallback: true
-},
+passport.use('foursquare', new OAuth2Strategy(
+  {
```

Comment:
Same as what @huttj mentioned - not sure about the line break.  Most of the rest of the project doesn't have such a line break before the '{'.  If eslint rules are causing this, we may want to look at why and if there is something in the eslint rules that need to be fixed.

Reply:
It doesn't look like that we have it configured and it is probably going to its default.  Perhaps we need to add `"function-paren-newline": ["error", "never"],` to the eslint configs.

doc: https://eslint.org/docs/rules/function-paren-newline